playground/resolve/__tests__/resolve.spec.ts

```diff
@@ -18,7 +18,7 @@ test('deep import with exports field', async () => {
 })
 
 test('deep import with query with exports field', async () => {
-  // since it is imported with `?url` it should return a url
+  // since it is imported with `?url` it should return an url
```

Comment:
```suggestion
  // since it is imported with `?url` it should return a URL
```

Similar to issue above with the unsounded `h`, use `a` before a sounded `u`.